process/README.md

```diff
@@ -0,0 +1,116 @@
+This policy describes the Cloud Native Computing Foundation (CNCF) project life cycle process, from sandbox to archival and more. It describes the stages a project may be admitted under and what the criteria and expectations are for a given stage, as well as the acceptance criteria for a project to move from one stage to another. It also describes the Annual Review Process through which those changes will be evaluated and made.
+
+Project progression, movement from one stage to another, allows projects to participate at the level that is most appropriate for them given where they are in their lifecycle.
+# II. Stages - Definitions & Expectations 
+CNCF projects have a maturity level of sandbox, incubating, or graduated. Archived is for projects no longer in active development. The maturity level is a signal by CNCF as to what sorts of enterprises should be adopting different projects. Projects increase their maturity by demonstrating their sustainability to CNCF’s Technical Oversight Committee: that they have adoption, a healthy rate of changes, committers from multiple organizations, have adopted the CNCF Code of Conduct, and have achieved and maintained the Core Infrastructure Initiative Best Practices Badge. 
+
+![Project Stages](https://github.com/cncf/toc/blob/master/process/project-stages.png)
+## Sandbox: 
+The CNCF Sandbox is the entry point for early stage projects and has four goals:
+
+* Encourage public visibility of experiments or other early work that can add value to the CNCF mission and build the ingredients of a successful Incubation level project
+* Facilitate alignment with existing projects if (and only if) this is desired
+* Nurture projects (e.g. via CNCF Service Desk requests)
+* Remove possible legal and governance obstacles to adoption and contribution by ensuring all projects adhere to CNCF legal, code of conduct and IP Policy requirements
+
+
+## Incubating: 
+Incubating projects have access to all resources listed at https://cncf.io/services-for-projects, and have a presence at CloudNativeCon. 
+
+
+## Graduated: 
+Graduated projects signal the highest level of maturity for a CNCF project.
+
+
+## Archived: 
+Archived projects are no longer in active development and are only archived after a TOC vote.
+
+# III. Project Proposal Process
+Introduction:
+This governance policy sets forth the proposal process for projects to be accepted into the CNCF.
+The process is the same for both existing projects which seek to move into the Foundation, and new projects to be formed within the CNCF.
+
+Project Proposal Requirements: 
+## Sandbox Projects: 
+Projects being submitted to the CNCF at the sandbox level are intended to be the entry point for early stage projects and are not required to undergo due diligence. A project is reviewed by a SIG (chosen by the TOC to review). After presenting to the SIG, a SIG will make a written recommendation to the TOC to review. Three sponsors are required for entry into the sandbox. 
```

Comment:
We might want to clarify that, while there is no TOC dilligence for proposed Sandbox projects, the appropriate SIG is expected to review the project before passing on a recommendation.

Reply:
There's a TOC triage step to make sure that it's at least appropriate for CNCF and outline which SIG to review.